source/concepts/what-is-ember-js.md

```diff
@@ -124,15 +124,18 @@ You can tell Ember about these dependencies like this:
 
 ```javascript
 var president = Ember.Object.create({
-  firstName: "Barack",
-  lastName: "Obama",
+  init: function() {
+    this._super.apply(this, arguments);
+    this.set('firstName', 'Barack');
```

Comment:
Is this kind of initialization encouraged? I thought that the *shape deoptimization* happened when defining attributes at a class level (using `Ember.Object.extend()`, but defining attributes with `create` was fine.